### PR TITLE
Rmacdon465

### PR DESCRIPTION
**Pull request by rmacdon465**
New pull request after killing the old one due to new branch.

### CODE REVIEW

File: tests/_support/AcceptanceTester.php [hunk 1]
```diff
@@ -68,6 +69,39 @@ public function generateEvent() {
...
+	 * Update Event
+	 */
+	public function edit_event( $updateEvent ) {
+		$originalTitle		= $updateEvent['originalTitle'];
+		$newTitle			= $updateEvent['newTitle'];
+		$content			= $updateEvent['content'];
+		$allDay 			= $updateEvent['allDay'];
+		if(empty($updateEventp['originalTitle']) || empty($updateEvent['newTitle'])){
+            throw new \InvalidArgumentException('Title missing');
+        }
```
Comment (borkweb) [inline tests/_support/AcceptanceTester.php]: There's some funky tabbing here with this `if` statement. Can you fix that?

**Comment by borkweb** [comment]
That one tabbing thing is the last item that should be resolved before this gets merged :) Sorry this has taken so long to review.

File: tests/_support/AcceptanceTester.php (hunk 1 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: Should be fixed!  Thanks

Comment (borkweb) [inline tests/_support/AcceptanceTester.php]: The funkiness is here:

```php
		$originalTitle			= $updateEvent['originalTitle'];
		$newTitle			= $updateEvent['newTitle'];
		$content			= $updateEvent['content'];
		$allDay 			= $updateEvent['allDay'];
		if(empty($updateEventp['originalTitle']) || empty($updateEvent['newTitle'])){
            throw new \InvalidArgumentException('Title missing');
        }
```

It should be:

```php
		$originalTitle			= $updateEvent['originalTitle'];
		$newTitle			= $updateEvent['newTitle'];
		$content			= $updateEvent['content'];
		$allDay 			= $updateEvent['allDay'];
		if ( empty( $updateEventp['originalTitle'] ) || empty( $updateEvent['newTitle'] ) ) {
			throw new \InvalidArgumentException( 'Title missing' );
		}
```

Comment (borkweb) [inline tests/_support/AcceptanceTester.php]: Hahaha...that looks the same, but it isn't :) The first block uses spaces before `throw new` and the last brace whereas the second block uses tabs.

Comment (borkweb) [inline tests/_support/AcceptanceTester.php]: #tabs4lyfe

**Comment by borkweb** [comment]
Let's merge this!

**Comment by borkweb** [comment]
:cake:

**Pull request merged**